console/src/console/Program.java

```diff
@@ -7,6 +7,7 @@ public static void main (String[] args) {
     System.out.println("Aaron is on the team!");
     System.out.println("We are an awesome team!");
     System.out.println("Greetings from Pam");
+    System.out.println("Im back");
```

Comment:
Welcome back! I'm going to leave this for another team member to review so that others can have the experience reviewing PRs :)